sormas-api/src/main/java/de/symeda/sormas/api/feature/FeatureTypeProperty.java

```diff
@@ -23,6 +23,7 @@ public enum FeatureTypeProperty {
 	ALLOW_FREE_EDITING(Boolean.class),
 	THRESHOLD_IN_DAYS(Integer.class),
 	EXCLUDE_NO_CASE_CLASSIFIED_CASES(Boolean.class),
+	MAX_CHANGE_DATE_SYNCHRONIZATION(Integer.class),
```

Comment:
```suggestion
	MAX_CHANGE_DATE_PERIOD(Integer.class),
```

(Just saw that this is how it's defined in the issue)